### PR TITLE
Code Review Project

### PR DESCRIPTION
**Pull request by gaks1**
1. Add HTML files.
2. Add CSS file.
3. Add javascript files.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@media screen and (max-width: 480px) {
+    main {
+        background-color: red;
+    }
+}
+
+#main-doc {
```
Comment (gaks1) [inline css/style.css:7]: I would  advice to use the css class instead of id. It  makes the code cleaner.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+}
+
+#left-part {
```
Comment (gaks1) [inline css/style.css:13]: I would advice to use the css class instead of id. It makes the code cleaner.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    width: 20%;
+    border-right: 1px solid black;
+    margin-right: 40px;
+}
+
+#right-part {
```
Comment (gaks1) [inline css/style.css:19]: I would advice to use the css class instead of id. It makes the code cleaner.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    width: 80%
+}
+
+#menu-list {
```
Comment (gaks1) [inline css/style.css:23]: I would advice to use the css class instead of id. It makes the code cleaner.

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    list-style-type: none;
+}
+
+#menu-list li a {
```
Comment (gaks1) [inline css/style.css:27]: I would advice to use the css class instead of id. It makes the code cleaner.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+          <hr />
+          <li><a href="#features" class="nav-link">Features</a></li>
+          <hr />
+          <li><a href="#syntax" class="nav-link">Syntax</a></li>
+          <hr />
+          <li><a href="#development_tools" class="nav-link">Development Tools</a></li>
+          <hr />
+        </ul>
+      </nav>
+    </div>
```
Comment (gaks1) [inline index.html:33]: Kindly use proper semantics. The best practice is to use the section tag for cleaner readable code.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+                    if(n2 >= n3)
+                        System.out.println(n2 + " is the largest number.");
+                    else
+                        System.out.println(n3 + " is the largest number.");
+                }
+            }
+        }
+        </code>
+      </section>
+    </div>
```
Comment (gaks1) [inline index.html:225]: Kindly use better semantics. Let the div element be a section element for readable and cleaner code.

File: js/index.js [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+    from red to green.
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
+
+// PLEASE NOTE: Adding global style rules using the * selector, or by adding rules to body {..} or html {..}, or to all elements within body or html, i.e. h1 {..}, has the potential to pollute the test suite's CSS. Try adding: * { color: red }, for a quick example!
+
+// Once you have read the above messages, you can delete all comments.
```
Comment (gaks1) [inline js/index.js:27]: I think you should remove unnecessary comments. The comments were to be deleted for cleaner code

**Review by gaks1 — COMMENTED** [review]
#Changes required ♻️ 

Make the changes as commented below.

Submit to rereview afterwards.

Thanks for the good work 😄

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @gaks1, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Comments with reasons ✔️

## Required Changes ♻️
_Check the comments under the review._
- [ ] I would advise you to make your comments politely, it could be preceded by 'i would advice', 'kindly', 'what do you think', 'I think' etc.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gaks1 — COMMENTED** [review]
#Changes Required ♻️ 

I think there are changes needed as commented  on the files.

Please make the necessary changes  and submit for a rereview. 

Thank you for the good work 😄

**Review by amejid — APPROVED** [review]
Hi @gaks1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Comments are polite :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**